package.json

```diff
@@ -29,7 +29,7 @@
     "conventional-changelog": "^1.1.0",
     "dateformat": "^1.0.11",
     "git-semver-tags": "^1.0.0",
-    "github": "^2.0.0",
+    "github": "^5.2.2",
```

Comment:
No, I believe 2.0.0 has the --draft option as well. We just wanted to use the latest version as good practice. I can change it back, though.